tornado_rest_client/utils.py

```diff
@@ -66,13 +68,14 @@ def populate_with_tokens(string, tokens, left_wrapper='%', right_wrapper='%',
     # First things first, swap out all instances of %<str>% with any matching
     # token variables found. If no items are in the hash (none, empty hash,
     # etc), then skip this.
-    allowed_types = (str, unicode, bool, int, float)
+    allowed_types = six.string_types + (six.text_type, str, bool, int, float)
+
     if tokens:
-        for k, v in tokens.iteritems():
+        for k, v in six.iteritems(tokens):
```

Comment:
i find `six.iteritems` to be really ugly.  unless you know a dict to be very large, and really need `iteritems()` i just use `items()` and move on with life.  in python3, `items()` is lazy like `iteritems()` is in python2